LLMonFHIR.xcodeproj/project.pbxproj

```diff
@@ -1175,6 +1188,22 @@
 				minimumVersion = 0.1.4;
 			};
 		};
+		979825512B79717200335095 /* XCRemoteSwiftPackageReference "SpeziLLM" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/StanfordSpezi/SpeziLLM";
+			requirement = {
+				branch = "feat/structural-improvments";
+				kind = branch;
+			};
+		};
+		979825562B79719B00335095 /* XCRemoteSwiftPackageReference "SpeziFHIR" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/StanfordSpezi/SpeziFHIR";
+			requirement = {
+				branch = "feat/lift-to-spezi-llm";
+				kind = branch;
+			};
+		};
```

Comment:
TODO to update them to the release versions once they have been merged and tagged.